pyuia/pageobject.py

```diff
@@ -143,8 +148,15 @@ def _assert_any_present(self, locators, handlers=None, check_visibility=False):
     def _assert_any_visible(self, locators, handlers=None):
         return self._assert_any_present(locators, handlers, check_visibility=True)
 
-    def _wait_present(self, locators, timeout_warn=_WARN_TIMEOUT, handlers=None,
-                      timeout=_WAIT_TIMEOUT, check_visibility=False):
+    def _wait_present(self, locators, timeout_warn=None, handlers=None,
+                      timeout=None, check_visibility=False):
+        """Defalt timeout_warn is assigned by _PAGE_WARN_TIMEOUT
+        Defalt timeout is assigned by _PAGE_WAIT_TIMEOUT"""
+        if not timeout_warn: 
+            timeout_warn = self._PAGE_WARN_TIMEOUT
```

Comment:
Per our discussion, there should be separate timeouts for waiting that is not related to page loading.